tensorflow/python/keras/applications/efficientnet_weight_update_util.py

```diff
@@ -0,0 +1,307 @@
+"""Utils for EfficientNet models for Keras.
+Write weights from  ckpt file as in original repo
+(https://github.com/tensorflow/tpu/tree/master/models/official/efficientnet)
+to h5 file for keras implementation of the models.
+"""
+
+import tensorflow as tf
+import tensorflow.keras as keras
+import h5py
+import numpy as np
+import argparse
+from tensorflow.keras.applications.efficientnet import *
```

Comment:
importing * seems to be a bad practice, can we explicitly list out all the needed functions/classes?

or you can do `from tensorflow.keras.applications import efficientnet`